docs/reference/release-notes/6.8.asciidoc

```diff
@@ -9,11 +9,35 @@ Also see <<breaking-changes-6.8,Breaking changes in 6.8>>.
 [float]
 === Bug fixes
 
+CCR::
+* Stop renew retention leases when follow task fails {es-pull}65168[#65168]
+
+Engine::
+* Fix the version and term field initialization error of `NoOpResult` {es-pull}66269[#66269] (issue: {es-issue}66267[#66267])
+
+Features/Monitoring::
+* Do not report negative values for swap sizes {es-pull}57317[#57317] (issues: {es-issue}42725[#42725], {es-issue}56435[#56435], {es-issue}57312[#57312])
```

Comment:
Not a big deal, but I think the other two issues here were just mentioned and so not worth including in the RN
```suggestion
* Do not report negative values for swap sizes {es-pull}57317[#57317] (issue: {es-issue}57312[#57312])
```